internal/propagation.go

```diff
@@ -81,6 +81,9 @@ func createOpenTracingSpan(
 	span := tracer.StartSpan(
 		name,
 		opentracing.StartTime(start),
+		// Sadly, the Otel BridgeTracer does not consider "FollowsFrom" as a valid reference
+		// for setting a parent SpanContext. A "ChildOf" reference is needed.
+		opentracing.ChildOf(parent),
 		opentracing.FollowsFrom(parent),
```

Comment:
huh.
being not familiar with the internals here: what's the difference?

Reply:
First, the necessity comes from this: https://github.com/open-telemetry/opentelemetry-go/blob/main/bridge/opentracing/bridge.go#L582

In terms of the Otel model, `follows-from` describes a case where the parent is not dependent on the child (I think the use-case in Cadence-land is running an activity but not blocking on the future, or perhaps a Child Workflow with an abandon `ParentClosePolicy`. `child-of` indicates that this dependency exists, [This might also give more context](https://github.com/w3c/trace-context/issues/202).